psq/queue_test.py

```diff
@@ -70,13 +81,23 @@ def testQueue(self):
 
             assert r.task_id == t.id
 
-    def testDequeue(self):
+    def test_dequeue(self):
         with patch('psq.queue.Queue._get_or_create_topic'):
             q = Queue(Mock())
             t = Task('1', sum, (1, 2), {'arg': 'c'})
             sub_mock = Mock()
             q._get_or_create_subscription = Mock(return_value=sub_mock)
 
+            # No messages
+            sub_mock.pull.return_value = []
+
+            tasks = q.dequeue()
+
+            assert sub_mock.pull.called
+            assert not tasks
```

Comment:
I actually don't know, but is it normal to use bult-in assert rather than unittest? I know for one thing -O strips them (although it's incredibly stupid to run tests with -O),  plus I thought built in ones gave better error messages. But, don't care, just curious why you picked this way.